### DISCUSSION TITLE
[Suggestion] Add fruits/vegetables and plants to Create: Crafts & Additions tags

### DISCUSSION THREAD

**Discussion by saltyseadoggo** [vectorwing/FarmersDelight#748] [Ideas]
As per the title, I am requesting that Farmer's Delight add its various fruits and vegetables to `#createaddition:plant_foods`, and its various plant items to `#createaddition:plants`. This will allow FD items to be used to make C:C&A biomass, without players/modpack devs needing to add all these items to these tags themselves.~
More specifically...

`#createaddition:plant_foods` should contain
`farmersdelight:cabbage`
`farmersdelight:tomato`
`farmersdelight:onion`
`farmersdelight:pumpkin_slice`

`#createaddition:plants` should contain
`farmersdelight:sandy_shrub`
`farmersdelight:wild_cabbages`
`farmersdelight:wild_onions`
`farmersdelight:wild_tomatoes`
`farmersdelight:wild_carrots`
`farmersdelight:wild_potatoes`
`farmersdelight:wild_beetroots`
`farmersdelight:wild_rice`
`farmersdelight:brown_mushroom_colony`
`farmersdelight:red_mushroom_colony`
`farmersdelight:rice_panicle`

**Comment by vectorwing** [comment]
I took a look at how C:C&A is handling its Biomass recipes, and it seems some of those are already covered in some recipes:
- The recipe asking for `forge:crops` accepts all FD crops, except for the Rice Panicle;
- The recipe asking for `minecraft:flowers` accepts all Wild Crops, as they're tagged as such;

This would leave a few of those out still, so we could still add this tag for them. I can take a look later. 👍

**Comment by vectorwing** [comment]
Pull request #719 was recently merged, with the suggested tags. Should be out in 1.2.2 soon!

**Comment by vectorwing** [comment]
1.2.2 has been released with this integration! 👍
